content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -3,14 +3,113 @@ title: Configure the Datadog Tracing Library
+**Supported Values**: A regex string <br>
+**Description**: Applies a regex to redact sensitive data from query strings on incoming HTTP requests. Matches are replaced with `<redacted>`. If an empty string is passed, no obfuscation occurs. The resulting value is reported in the `http.url` tag. 
+
+### Diagnostics
+
+`DD_TRACE_LOG_DIRECTORY`
+: **Default**: Varies by SDK, environment, and runtime <br>
+**Supported Values**: A valid directory path that exists on the system <br>
+**Not Supported In**: Node.js, Ruby, Python SDKs<br>
+**Description**: Specifies the directory where tracer log files should be routed. If the directory does not exist, the SDK logs a warning and behaves as if the environment variable was not set.
+
+### Agent
+
+`DD_TRACE_AGENT_URL`
+: **Default**: `http://localhost:8126` <br>
+**Supported Values**: A string representing a host name <br>
+**Description**: The URL for connecting the tracer to the Datadog agent. Valid URL schemas include `http://` and `unix://` (Unix Domain Sockets). This value takes precedence over `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` if set.
+
+### Unified Service Tagging
+
+`DD_VERSION`
+: **Default**: `null` <br>
+**Supported Values**: A string representing an application version <br>
+**Description**: Adds a version tag to all spans that have the same service name as the global service name (aka `DD_SERVICE`). Spans with a different service name do not get this tag.
+
+`DD_SERVICE`
+: **Default**: `null` <br>
+**Supported Values**: A string representing an application service name <br>
+**Description**: Sets the default service name used for most spans. In some cases, tracers may set a different service name for downstream services.
+
+`DD_ENV`
+: **Default**: `null` <br>
+**Supported Values**: A string representing an application environment name (e.g., `prod`, `dev`) <br>
+**Description**: Adds an environment tag to all spans generated by the tracer instance.
+
+### Integrations
+
+`DD_TRACE_HTTP_CLIENT_ERROR_STATUSES`
+: **Default**: `400-499` <br>
+**Supported Values**: A comma-separated string of the form `from-to`, where `from` and `to` are inclusive integers. Singular values are also accepted (e.g., `400-403,405,410-499`). <br>
+**Not Supported In**: Node.js SDK<br>
+**Description**: Defines the range of status codes to be considered as errors on `http.client` span kinds. Only the values within the specified range are considered errors.
```

Comment:
What do you think of the following suggestion? Since we don't use the same name of `http.client` everywhere, I think we should avoid using the code markdown so we don't create confusion. This feedback also applies to the server statuses configuration.
```suggestion
**Description**: Defines the range of status codes to be considered as errors on automatically collected HTTP client spans. Only the values within the specified range are considered errors.
```